src/components/CopyButton/index.tsx

```diff
@@ -44,7 +44,7 @@ const StyledButton = styled('button', {
   }
 `
 
-type CopyButtonProps = {
+export type CopyButtonProps = {
```

Comment:
Use ComponentProps instead

Reply:
To get props from where?

Reply:
I think he mean don't export the type, and when you want to use it in another file, use `ComponentProps<typeof CopyButton>` @matthprost

Reply:
@QuiiBz Ahh okay makes sense now